docs/recipes/close-account.md

```diff
@@ -0,0 +1,30 @@
+# Close Account
+
+In accounting, a _closing entry_ zeroes a (temporary) account's balance.
+
+### Example
+
+Given a set of accounts:
+
+| Account | Debits Pending | Debits Posted | Credits Pending | Credits Posted | Flags                            |
+| ------: | -------------: | ------------: | --------------: | -------------: | -------------------------------- |
+|     `A` |              0 |            10 |               0 |             20 | `debits_must_not_exceed_credits` |
+|     `B` |              0 |            20 |               0 |             10 | `credits_must_not_exceed_debits` |
```

Comment:
Can we make this more tricky so that the accounts have different net balances? e.g. One can have 10 and the other can have 5.

Reply:
So that none of the example values repeat...